freqtrade/plot/plotting.py

```diff
@@ -96,20 +96,34 @@ def add_indicators(fig, row, indicators: Dict[str, Dict], data: pd.DataFrame) ->
                        Dict key must correspond to dataframe column.
     :param data: candlestick DataFrame
     """
+    plot_kinds = {
+        'scatter': go.Scatter,
+        'bar': go.Bar,
+    }
     for indicator, conf in indicators.items():
         logger.debug(f"indicator {indicator} with config {conf}")
         if indicator in data:
             kwargs = {'x': data['date'],
                       'y': data[indicator].values,
-                      'mode': 'lines',
                       'name': indicator
                       }
-            if 'color' in conf:
-                kwargs.update({'line': {'color': conf['color']}})
-            scatter = go.Scatter(
-                **kwargs
-            )
-            fig.add_trace(scatter, row, 1)
+
+            plot_type = conf.get('type', 'scatter')
+            color = conf.get('color')
+            if plot_type == 'bar':
+                kwargs.update({'marker_color': color or 'DarkSlateGrey',
+                               'marker_line_color': color or 'DarkSlateGrey'})
+            else:
+                if color:
+                    kwargs.update({'line': {'color': color}})
+                kwargs['mode'] = 'lines'
+                if plot_type != 'scatter':
+                    logger.warning(f'Indicator {indicator} has hnknown plot trace kind {plot_type}'
+                                   f', assuming "scatter".')
+
+            kwargs.update(conf.get('kwargs', {}))
```

Comment:
i do have one problem with using "kwargs" here.

I don't see plotting with plotly as the future for freqtrade - and the arguments needed for echarts (freqUI) will be different.
All options we have now (except for Area plots) are supported by both plotly (plot modules) and freqUI with the same arguments - using kwargs would be a "pure" plotly function - which would make the plot-config no longer work for freqUI (and might even cause odd side-effects when loading the config from the strategy there).

Reply:
Ah i did not think of it. Could name it `plotly_args` to disambiguate. Would that work? Or maybe just `plotly`?

Reply:
yeah - maybe 

if the docs (!) contain a hint that these options are only supported for plotly plotting (not for FreqUI) - i guess it'll be fine